### ISSUE TITLE
confluent-3.2.0 S3 Connector default partition settings

### ISSUE THREAD

**Issue by warsameahmed** [confluentinc/kafka-connect-storage-cloud#63]
I have been able to successfully write my topic date stream to my s3 bucket, however, the data is partition numerically (partition=0).

current display: (Amazon S3 > bucket-name > topics > test > partition=0

Is it possible to partition the data by Date(yyyy-mm-dd). And if so where would be the configuration to override the default partition settings.

Thank you for your help,
Warsame

**Comment by kkonstantine** [comment]
For a fully featured time-based partitioning you will want to checkout `master` or `3.2.x` branches (also tag `v3.2.2`) if you are compiling from source. 

Additionally it comes along with Confluent Platform 3.2.2 ([release notes](http://docs.confluent.io/current/release-notes.html#s3-connector)) and the upcoming 3.3.0 release.

**Comment by warsameahmed** [comment]
Thanks @kkonstantine I mean to say confluent-3.2.0. I have this version running. Just don't know where to update the setting.

thnx

**Comment by kkonstantine** [comment]
You will have to upgrade at least to `3.2.2` from github or https://www.confluent.io/download/

**Comment by warsameahmed** [comment]
Okay thanks @kkonstantine.

**Comment by ewencp** [comment]
@warsameahmed You'd want to change the [partitioner.class configuration](http://docs.confluent.io/current/connect/connect-storage-cloud/kafka-connect-s3/docs/configuration_options.html#partitioner). Did you sort this out or is there any other follow up for this issue?

**Comment by warsameahmed** [comment]
@ewencp thanks for the followup. Yes I was able to resolve this by setting the partitioner class TimeBasedPartitioner rather than the Default. However now this introduced another problem. 

Before upgrading to 3.2.2., in my previous version 3.2.0 I was able to start connect standalone and pass the source jar file (twitter4j) and the s3 sink properties. This was an issue I was having earlier which I resolved, but now since the upgrade to 3.2.2 it's back again.

When will the multiple SLF4J implementations in the classpath (It looks like there are 3: 1.7.5, 1.7.6, 1.7.21) be resolved.  I think this causing some issues.
